src/js/components/Stats.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import {AxiosInstance as axios} from "axios";
+
+class Stats extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            trending_hashtags: [],
+            num_daily_posts: [],
+        }
+
+    }
+    componentDidMount() {
+        this.getTrendingHashtags();
+    }
+
+    getTrendingHashtags = () => {
+        axios.get(`${process.env.REACT_APP_API_URL}` + 'stats/trending').then(
+            response => {
+                this.setState({trending_hashtags: response.data})
+            }
+        )
+    };
+
+    render() {
+        return (
+            <h1>Stats</h1>
```

Comment:
Should display Chart here using methods from ```react-google-charts```

Reply:
I  know my dude. I don't understand :(

Reply:
Try using a table chart to make something similar to this. Without the get report thing 

![Screenshot_2019-04-24 The Complete Guide to Hashtag Analytics](https://user-images.githubusercontent.com/22486123/56632227-c7e56200-6626-11e9-905f-c54a0d77af49.png)